core/amber/src/main/python/pytexera/udf/examples/rudf/r_tuple_operator.py

```diff
@@ -0,0 +1,56 @@
+# Note: make sure R path is initialized in udf.conf and make sure that
+# the following packages (in R) are installed: coro, arrow
```

Comment:
how do we ensure those packages are installed correctly? from https://github.com/r-lib/actions/tree/v2/setup-r-dependencies I see one way is to declare dependencies needs to install in `DESCRIPTION` file. if so, can you try come up with a new `DESCRIPTION` file at the same folder where we keep Python's `requirements.txt`, so that the CI can pick up the proper R dependencies?

Reply:
more to read about the `DESCRIPTION` file: https://r-pkgs.org/description.html

Reply:
I was able to add a `DESCRIPTION` file to allow GitHub's Workflow Actions to automatically install all R libraries listed in the `DESCRIPTION`. I think the user/developer will still have to manually install the libraries, so they will have to read the `DESCRIPTION` file themselves. Will continue to see if I can find a solution.